Remove status default from README

sindrenm opened the pull request:
The “status” input is required to specify and does not have a default, so remove “defaults to `completed`” from its description.

boswelja commented:
Except it does default to `completed`
https://github.com/r0adkll/upload-google-play/blob/d5cf6e94453c4037a275bc9a151d44f1f7f134f8/action.yml#L40

sindrenm commented:
So it would seem, but what about its description then?

> Defaults to `completed` if targeting 100% rollout, `halted` for 0%, and `inProgress` for anything in between.

Also, if it's marked as required, I would guess it was required to pass it in scripts, which, if it has a default, doesn't make sense. Perhaps I'm just misunderstanding something here, I just find the current combination of description, default and required being true to be confusing.

boswelja commented:
True, the rest of the description is incorrect. The idea of marking it as required with a default parameter is to (hopefully) avoid people passing null, though I'm not sure that's even possible 😅

sindrenm commented:
Right, that makes sense, except I feel like that should be covered by its “Value”:

> One of completed, inProgress, halted, draft

I can change this PR to remove the erroneous description and possibly set “Required” to false if you want, or we can close this and make another. Which do you prefer? :slightly_smiling_face:

boswelja commented:
I'm happy for both to be done in one PR if that's easiest for you

sindrenm commented:
I've updated this PR now. Added a “cannot be null” text to its value, although I personally feel like it's a bit redundant because of the already mentioned allowed values. Nevertheless, it doesn't hurt to have it there.
